WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -19553,7 +19541,7 @@
 				bg,
 				sk,
 			);
-			mainGroup = 29B97314FDCFA39411CA2CEA;
```

Comment:
> Includes some Xcode project cleanup noise

I think it's coming from your Xcode version. Have you updated to Xcode 15.4? A PR landed recently https://github.com/wordpress-mobile/WordPress-iOS/pull/23259, but I'm still on 15.1.

Reply:
Oh, I see that we now fully switched to 15.4 – https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/.xcode-version. I tried it and I'm getting the same diff after making any pbxproj changes. Resolving.